src/confdb/confdb.c

```diff
@@ -2438,3 +2456,146 @@ int confdb_certmap_to_sysdb(struct confdb_ctx *cdb,
 
     return ret;
 }
+
+/**
+ * Retrieve the list of enabled domains considering the explicit list
+ * and the new 'enabled' attribute.
```

Comment:
Don't use 'new' here since this option will not remain new for the eternity :-)

Reply:
hehe! right, thank you :) Done